### DISCUSSION TITLE
Make JsonSerialization Case Insensitive

### DISCUSSION THREAD

**Discussion by taooceros** [Flow-Launcher/Flow.Launcher#546] [Q&A] [answered]
Should we make JsonSerialization Case Insensitive for JsonRpcPlugin?

Currently, there are two naming policy for our python plugin. The outer response model is camel case, but the inner Result (JsonRPCResult which extends the Result class) is Pascal Case (first letter Upper). Should we mark all camel case or simply use Case Insensitive.

Any thought? @Flow-Launcher/team

**Comment by jjw24** [comment, marked as answer]
i think use case insensitive, coming from the perspective that it makes it easier for py devs. Python and C# use different naming conventions anyway.
